docs/_data/components/schemas/limit-provenance.yaml

```diff
@@ -1,63 +1,75 @@
-type: object
-description: Given a limit value, a set of data defining the provenance of that limit.
-properties:
-  proposals-considered:
-    description: |
-
-      The forecast proposals provided by the Ratings Providers during the
-      Forecast Window for this limits snapshot.
-
-    type: array
-    maxItems: &max-proposals 10
-    items:
-      allOf:
-        - $ref: 'forecast-limit-period.yaml#/proposal-considered'
-        - type: object
-          properties:
-            resource-id:
-              $ref: ./resource-id.yaml
-
-  temporary-aar-exceptions:
-    description: |
-
-      The temporary AAR exceptions for the facility that were active when this
-      snapshot was generated.
-
-    type: array
-    maxItems: *max-proposals
-    items:
-      $ref: ./resource-id.yaml
-
-  overrides:
-    type: array
-    minItems: 0
-    maxItems: *max-proposals
-    items:
-      type: object
-      properties:
-        override:
-          $ref: ./limit-data.yaml
-        override-reason:
-          type: string
-          description:
+forecast: &forecast
+  type: object
+  description: Contains the provenance data for a given limit.
+  properties: &props
+    proposals-considered: &considered
+      description: |
+
+        The ratings proposals provided by the Ratings Providers that were
+        considered when determining the limits of the facility for a specific
+        period.
+
+        If `resource-id` is not provided for a proposal, the ratings proposal is
```

Comment:
@getorymckeag this PR is mostly a yaml refactoring so that I could get this text to show up in both Forecast and RT detailed limits snapshots.